ci/codebuild/amazonlinux-2017.03.yml

```diff
@@ -9,9 +9,9 @@ phases:
   build:
     commands:
       - echo Build started on `date`
-      - ci/codebuild/build.sh -DENABLE_TESTS=ON
-      - ci/codebuild/run-tests.sh  aws-lambda-package-lambda-test-fun
-      - ci/codebuild/run-tests.sh  aws-lambda-package-lambda-test-fun-no-glibc
+      - ci/codebuild/build.sh -DENABLE_TESTS=ON -DTEST_RESOURCE_PREFIX=amzn201703
```

Comment:
Worth adding amzn201803 tests as well. To go along with https://aws.amazon.com/blogs/compute/upcoming-updates-to-the-aws-lambda-execution-environment/

Reply:
Definitely. This PR sets up the CI to accommodate for multiple platforms. I'm just starting with Ubuntu.